torax/geometry_loader.py

```diff
@@ -58,6 +59,11 @@ def _load_fbt_data(file_path: str) -> dict[str, np.ndarray]:
   """Loads the data from a FBT-LY file into a dictionary."""
   return scipy.io.loadmat(file_path, squeeze_me=True)
 
+from eqdsk import EQDSKInterface
+def _load_eqdsk_data(file_path: str) -> dict[str, np.ndarray]:
+    eq = EQDSKInterface.from_file(file_path, no_cocos=True) # should probably handle COCOS shenanigans...
```

Comment:
Is this something we need to handle here? Or maybe turn it into a formal todo

Reply:
This needs a formal TODO, not sure how the torax team wants to handle COCOS in the future.